OutOfSchool/OutOfSchool.WebApi.Tests/Services/Database/WorkshopServiceTests.cs

```diff
@@ -405,36 +403,36 @@ public void UpdateStatus_WhenEntityIsInvalid_ShouldReturn_ArgumentException()
         workshopRepository.VerifyAll();
     }
 
-    [Test]
-    public async Task Update_WhenTryUpdateStatus_ShouldReturnEntityWithOldStatus([Random(1, 100, 1)] long classId)
-    {
-        // Arrange
-        var provider = ProvidersGenerator.Generate();
-        var inputWorkshopDto = WithWorkshop(Guid.NewGuid());
-        inputWorkshopDto.Status = WorkshopStatus.Closed;
-        inputWorkshopDto.Provider = provider;
-        var expectedStatus = WorkshopStatus.Open;
-        var workshopDtoMock = WithWorkshop(Guid.NewGuid());
-        workshopDtoMock.Provider = provider;
-
-        workshopRepository.Setup(w => w.GetWithNavigations(It.IsAny<Guid>())).ReturnsAsync(workshopDtoMock);
-        workshopRepository.Setup(w => w.UnitOfWork.CompleteAsync()).ReturnsAsync(It.IsAny<int>());
-        mapperMock.Setup(m => m.Map<WorkshopDTO>(workshopDtoMock))
-            .Returns(mapper.Map<WorkshopDTO>(workshopDtoMock));
+    //[Test]
+    //public async Task Update_WhenTryUpdateStatus_ShouldReturnEntityWithOldStatus([Random(1, 100, 1)] long classId)
+    //{
+    //    // Arrange
+    //    var provider = ProvidersGenerator.Generate();
+    //    var inputWorkshopDto = WithWorkshop(Guid.NewGuid());
+    //    inputWorkshopDto.Status = WorkshopStatus.Closed;
+    //    inputWorkshopDto.Provider = provider;
+    //    var expectedStatus = WorkshopStatus.Open;
+    //    var workshopDtoMock = WithWorkshop(Guid.NewGuid());
+    //    workshopDtoMock.Provider = provider;
 
-        mapperMock.Setup(m => m.Map<List<DateTimeRange>>(It.IsAny<List<DateTimeRangeDto>>()))
-            .Returns(mapper.Map<List<DateTimeRange>>(It.IsAny<List<DateTimeRangeDto>>()));
+    //    workshopRepository.Setup(w => w.GetWithNavigations(It.IsAny<Guid>())).ReturnsAsync(workshopDtoMock);
+    //    workshopRepository.Setup(w => w.UnitOfWork.CompleteAsync()).ReturnsAsync(It.IsAny<int>());
+    //    mapperMock.Setup(m => m.Map<WorkshopDTO>(workshopDtoMock))
+    //        .Returns(mapper.Map<WorkshopDTO>(workshopDtoMock));
 
-        workshopRepository.Setup(r => r.RunInTransaction(It.IsAny<Func<Task<Workshop>>>()))
-            .Returns((Func<Task<Workshop>> f) => f.Invoke());
+    //    mapperMock.Setup(m => m.Map<List<DateTimeRange>>(It.IsAny<List<DateTimeRangeDto>>()))
+    //        .Returns(mapper.Map<List<DateTimeRange>>(It.IsAny<List<DateTimeRangeDto>>()));
 
-        // Act
-        var result = await workshopService.Update(mapper.Map<WorkshopDTO>(inputWorkshopDto)).ConfigureAwait(false);
+    //    workshopRepository.Setup(r => r.RunInTransaction(It.IsAny<Func<Task<Workshop>>>()))
+    //        .Returns((Func<Task<Workshop>> f) => f.Invoke());
 
-        // Assert
-        workshopRepository.VerifyAll();
-        Assert.AreEqual(expectedStatus, result.Status);
-    }
+    //    // Act
+    //    var result = await workshopService.Update(mapper.Map<WorkshopBaseDTO>(inputWorkshopDto)).ConfigureAwait(false);
```

Comment:
(Commented out code) Please consider if this whole test should be present here. My suggestion is to either uncomment and fix it or remove it.
Because now is the moment, when commented code is no longer valid, and not supported, because of previous changes. 
I.e. commented code contains
- `WorkshopBaseDTO` that now becomes `WorkshopBaseDto`, 
- `WorkshopDTO` (mentioned earlier in `mapperMock.Setup`) could be `WorkshopBaseDto` as well,
- the whole `mapperMock.Setup` block could probably be removed by `SetupUpdate()` (though I'm not sure).

Reply:
Fixed it. The redundant test was removed.